tf-modules/archive/variables.tf

```diff
@@ -147,7 +147,7 @@ variable "buckets" {
 }
 
 variable "cmr_custom_host" {
-  description = "Custom host to use for CMR requests"
+  description = "Custom protocol and host to use for CMR requests (e.g. http://cmr-host.com)"
   type        = string
   default     = ""
```

Comment:
Should this be null? Can we also reference this in `tf-modules/cumulus/archive.tf` now?